pkg/cmd/status/status.go

```diff
@@ -0,0 +1,616 @@
+package status
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"net/http"
+	"net/url"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/charmbracelet/lipgloss"
+	"github.com/cli/cli/v2/api"
+	"github.com/cli/cli/v2/internal/ghinstance"
+	"github.com/cli/cli/v2/pkg/cmdutil"
+	"github.com/cli/cli/v2/pkg/iostreams"
+	"github.com/cli/cli/v2/utils"
+	"github.com/spf13/cobra"
+)
+
+type StatusOptions struct {
+	HttpClient   func() (*http.Client, error)
+	CachedClient func(*http.Client, time.Duration) *http.Client
+	IO           *iostreams.IOStreams
+	Org          string
+	Exclude      string
+}
+
+func NewCmdStatus(f *cmdutil.Factory, runF func(*StatusOptions) error) *cobra.Command {
+	opts := &StatusOptions{
+		CachedClient: func(c *http.Client, ttl time.Duration) *http.Client {
+			return api.NewCachedClient(c, ttl)
+		},
+	}
+	opts.HttpClient = f.HttpClient
+	opts.IO = f.IOStreams
+	cmd := &cobra.Command{
+		Use:   "status",
+		Short: "Print information about relevant issues, pull requests, and notifications across repositories",
+		Long: heredoc.Doc(`
+			The status command prints information about your work on GitHub across all the repositories you're subscribed to, including:
+
+			- Assigned Issues
+			- Assigned Pull Requests
+			- Review Requests
+			- Mentions
+			- Repository Activity (new issues/prs, comments)
+		`),
+		Example: heredoc.Doc(`
+			Exclude some repositories:
+
+			gh status -e"cli/cli,cli/go-gh"
+
+			Limit information to a single organization's repositories:
+
+			gh status -ocli
+		`),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if runF != nil {
+				return runF(opts)
+			}
+
+			return statusRun(opts)
+		},
+	}
+
+	cmd.Flags().StringVarP(&opts.Org, "org", "o", "", "Report status within an organization")
+	cmd.Flags().StringVarP(&opts.Exclude, "exclude", "e", "", "Comma separated list of repos to exclude in owner/name format")
```

Comment:
This could be a StringSliceVar saving to a `[]string` field. That way the caller could invoke `-e` multiple times and aggregating all those values in a single slice would be handled by the flags processor.